eclipse-theia/theia
# test: fix deprecation warnings in tests

#### What it does


The pull-request includes the following updates:

1. updates the tests to fix deprecation and other warnings
2. updates the `ext:build` script to reduce `verbosity`
3. updates the repo's `build` script to reduce `verbosity`.

The verbosity during the build was not useful as it never displayed any relevant information when building individual extensions and would ultimately pollute the log. The verbosity is now lowered, and only relevant information such as `errors` and `warnings` will now be displayed. This also helps navigating the log during CI.

#### How to test


1. verify that the build successfully passes during CI
2. verify that the tests successfully pass during CI
3. verify that the overall deprecation warnings from tests has been fixed.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
